modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/metastorage/MetaStorage.java

```diff
@@ -85,6 +86,15 @@
     /** */
     public static final int METASTORAGE_CACHE_ID = CU.cacheId(METASTORAGE_CACHE_NAME);
 
+    /** Metastorage cache directory to store data. */
+    public static final String META_STORAGE_DIR_NAME = "metastorage";
```

Comment:
I suggest renaming it to ``METASTORAGE_DIR_NAME`` (similar to other constants)

Reply:
Fixed.